packages/template-retail-react-app/app/pages/cart/index.test.js

```diff
@@ -154,7 +196,8 @@ test('Applies default shipping method to basket and renders estimated pricing',
 })
 
 describe('Update quantity', function () {
-    test.skip('Can update item quantity in the cart', async () => {
+    test('Can update item quantity in the cart', async () => {
+        jest.setTimeout(30000)
```

Comment:
my guess is that this is related to modals? have we taken a deeper look at modals and why do we need to setTimeout?

Reply:
We can try to use timeout on the waitFor below instead of exclusive set jestTimeout here

Reply:
Tests are passing when I use timeout on the `waitFor` in this cart test, but for the modal test, the jestTimeout is necessary